map_gen/maps/crash_site/commands.lua

```diff
@@ -570,6 +570,14 @@ function Public.control(config)
         end
     end
 
+    function Public.call_strike(args, player)
```

Comment:
In cases like this you can just do:
```suggestion
    Public.call_strike = strike
```

Reply:
Fixed